### ISSUE TITLE
FR: Docker container (on installation page)

### ISSUE THREAD

**Issue by rugk** [tailscale/tailscale#8776]
### What are you trying to do?

I am a new user to Tailscale. I just signed up with SSO.

I just got to this page (similar):
![image](https://github.com/tailscale/tailscale/assets/11966684/1baadead-1bec-491e-9d48-c5555bae3f67)

I don't want to install any packages on my system as they pollute it and are not really separated/isolated and my distribution (Fedora SIlverblue) does not even allow to install rpm's etc. the "traditional way". As such, I want an alternative.

### How should we solve this?

You could provide a Docker container that you can run with Docker, podman etc.

The installation guide can then also drastically be simplified by just running `docker-compose up` e.g. if you provide a compose file or even a single container.

### What is the impact of not solving this?

Containers are a big famous deployment mechanism that is surely getting more and more popular. Also they can provide one of the simplest installation/setup mechanisms possible.

And for a user who "just wants to try things out" they are excellent as you can easily remove them and no traces or stuff will be left on your system/leaving your system in a clean state.

As such, if you don't do this, you may loose many users as me, who are not keen on running random shell scripts or installing random packages from your own package registries just to try things out.

### Anything else?

I searched this issue tracker and I was confused, because some issues talk about docker, so maybe you already provide one? If so, it is just not linked on that page, which decreases discoverability by a huge amount, so I – as a new user – cannot find it.

**Comment by DentonGentry** [comment]
Does https://hub.docker.com/r/tailscale/tailscale do what you need?

**Comment by rugk** [comment]
Yes, then take this as a request to add it to the docs/installation guide, I wonder why it is not there…
